Image_Assets/optimise_images.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#   optimise_images.py - Call convert on PNGs and JPEGs using Google's settings
+#   Requires: Python >= 3.6
+#   Requires: Dank memes in production.
+
+from subprocess import CalledProcessError, check_output  # noqa: S404
+from sys import argv, exit
+
+ARG = "--replace"
+_LEN = len(argv)
```

Comment:
Is the prefix here to avoid almost shadowing `len`? I think this is fine but consider `argc` (in any capitalisation) as I think that will make the purpose of this variable more obvious.

Reply:
Oh my God, yes! :laughing: It turns out that `argc` exists and has existed for decades :1st_place_medal: